README.md

```diff
@@ -387,53 +400,70 @@ Unit Testing
 Utilities
 =========
 
-This contains anything which doesn't fit into another category.
+Language extensions
+-------------------
 
+* [cl21][125] - An experimental project to move Common Lisp into the 21st century. [Expat][14].
 * [alexandria][149] - A general-purpose utility library. Public domain.
 * [anaphora][158] - A collection of anaphoric macros. Public domain.
+* [cl-algebraic-data-type][146] - A library for defining algebraic data types in a similar spirit to Haskell or ML. [3-clause BSD][15].
+* [FSet][164] - A functional, set-theoretic collections data structure library. [LLGPL][8].
+* [iterate][139] - An iteration construct for Common Lisp which is extensible and Lispier. No license specified.
+* [optima][146] - Optimized pattern-matching library. [LLGPL][8].
+* [trivial-types][145] - Trivial type definitions. [LLGPL][8].
+* [interface][148] - A protocol library. [3-clause BSD][15].
+
+Files and directories
+---------------------
+
+* [cl-fad][111] - A portable pathname library. [FreeBSD][39].
```

Comment:
I think we should abandon `cl-fad ` in favor of `uiop`, but that's just my two cents.